packages/reactive_ble_mobile/android/src/main/kotlin/com/signify/hue/flutterreactiveble/ble/ReactiveBleClient.kt

```diff
@@ -152,24 +154,31 @@ open class ReactiveBleClient(private val context: Context) : BleClient {
         }.firstOrError()
     }
 
+    @SuppressLint("MissingPermission")
```

Comment:
I recommend against adding `@SuppressLint("MissingPermission")` to the library in order to suppress lint warnings. Instead, we have provided clear instructions in the README about the required permissions for the library's proper functioning. It's important to maintain a transparent approach to permissions and ensure that developers integrating the library are aware of and handle the necessary permissions appropriately.

Reply:
How does adding a `SuppressLint` conflict with "maintain[ing] a transparent approach to permissions and ensur[ing] that developers integrating the library are aware of and handle the necessary permissions appropriately"?